akka-sample-cqrs-scala/src/main/scala/sample/cqrs/Settings.scala

```diff
@@ -0,0 +1,43 @@
+package sample.cqrs
+
+import akka.actor.{Actor, ExtendedActorSystem, Extension, ExtensionId, ExtensionIdProvider}
+import com.typesafe.sslconfig.util.EnrichedConfig
+
+import scala.concurrent.duration.FiniteDuration
+
+object Settings extends ExtensionId[SettingsImpl] with ExtensionIdProvider {
+  override def lookup = Settings
+
+  override def createExtension(system: ExtendedActorSystem) = new SettingsImpl(system)
+}
+
+class SettingsImpl(system: ExtendedActorSystem) extends Extension {
```

Comment:
rename to `Settings`

(it's the public api of the extension)

Reply:
Correct